events/test.py

```diff
@@ -40,21 +41,21 @@ def setUp(self):
         self.test_info = {'name': 'test_event',
                           'description': 'testing!',
                           'author': 'admin',
-                          'event_time': '7-12-2019',
-                          'created_at': '7-10-2019'}
+                          'event_time': example_time,
+                          'created_at': example_time}
         self.test_info_with_id = dict(event_id=1, **self.test_info)
         self.test_info_with_db_id = dict(event_id=1, _id=2, **self.test_info)
 
     def test_construct_event(self):
         event = app.Event(self.test_info)
         self.test_info['event_id'] = None
```

Comment:
Doesn't your `Event` constructor automatically set `'event_id'` to `None` if it's not in the info?

Reply:
I wanted to test cases when event info had been queried, possibly after being re-entered, from the DB (so it would have an ID defined) in addition to when it was being user-supplied.

Reply:
Sorry, I'm a little confused. So for this first assert from lines 50-52, you're comparing `test_info` with an `Event` constructed from `test_info`, right? It seems to me that setting `self.test_info['event_id'] = None` shouldn't be necessary since your `Event.__eq__()` ignores `event_id`, no?

Reply:
I'm actually comparing the info stored by the event object constructed from `self.test_info` with `self.test_info` itself. Since the constructor adds sets `event_id = None`, I have to do the same thing for `self.test_info`

Reply:
Ah got it now, sorry for my confusion.